Spigot-Server-Patches/0345-Mob-Spawner-API-Enhancements.patch

```diff
@@ -82,10 +82,29 @@ index c3a5db97fd85b31c6b4bce93527b9d0e16a79a45..c2e830db7aa2944a477624e149a3ba66
              nbttagcompound.setShort("MaxNearbyEntities", (short) this.maxNearbyEntities);
              nbttagcompound.setShort("RequiredPlayerRange", (short) this.requiredPlayerRange);
 diff --git a/src/main/java/org/bukkit/craftbukkit/block/CraftCreatureSpawner.java b/src/main/java/org/bukkit/craftbukkit/block/CraftCreatureSpawner.java
-index 5c4c3c70c7450a3e556d664a1bf6bcaf8e3a571c..e78e3804bad9b0b6004939cadf40465dec68fdba 100644
+index 5c4c3c70c7450a3e556d664a1bf6bcaf8e3a571c..bc0dc763f8c95e697b05d4c003dda57fd1c59c58 100644
 --- a/src/main/java/org/bukkit/craftbukkit/block/CraftCreatureSpawner.java
 +++ b/src/main/java/org/bukkit/craftbukkit/block/CraftCreatureSpawner.java
-@@ -121,4 +121,16 @@ public class CraftCreatureSpawner extends CraftBlockEntityState<TileEntityMobSpa
+@@ -2,12 +2,18 @@ package org.bukkit.craftbukkit.block;
+ 
+ import com.google.common.base.Preconditions;
+ import net.minecraft.server.EntityTypes;
++import net.minecraft.server.IRegistry;
```

Comment:
do we want to add all these imports?

Reply:
Might as well just move them to their own block to avoid conflicts as much as possible? Fully qualified names are fine too, but I'd argue it's much more readable with imports :p

Reply:
yeah, one big block is prolly fine